cmd/build/command.go

```diff
@@ -155,6 +156,11 @@ func (bc *Command) getBuilder(options *types.BuildOptions, okCtx *okteto.OktetoC
 			return nil, err
 		}
 
+		var userFriendlyErr *suggest.UserFriendlyError
```

Comment:
I've added this check to avoid proceeding with a fallback to v1. Without this change we would return a misleading error: `Invalid Dockerfile: 'Dockerfile' is not a regular file`. Now we return `manifest validation failed: service 'frontend' build section not defined correctly`.